src/scenes/Root/Header/ProfileToolbar/ProfileToolbar.tsx

```diff
@@ -30,13 +23,13 @@ export const ProfileToolbar = () => {
   const { session } = useSession();
   const impersonation = useContext(ImpersonationContext);
   const [profileAnchor, setProfileAnchor] = useState<MenuProps['anchorEl']>();
-  const [actionsAnchor, setActionsAnchor] = useState<MenuProps['anchorEl']>();
 
   return (
     <>
-      <Card className={classes.card}>
+      <div className={classes.card}>
         <Typography className={classes.name} color="primary">
```

Comment:
```suggestion
        <Typography color="primary" sx={{ fontWeight: 'medium' }}>
```
And then the useStyles can be gotten rid of.